gradle.properties

```diff
@@ -2,10 +2,10 @@ org.gradle.caching=true
 org.gradle.parallel=true
 org.gradle.jvmargs='-Dfile.encoding=UTF-8'
 org.gradle.workers.max=3
-version=2201.1.1-SNAPSHOT
+version=2201.1.0-rc1.5-SNAPSHOT
```

Comment:
Version should be `2201.1.1-SNAPSHOT`. right? @lasinicl

Reply:
My bad. Will send a PR to fix this!